figures/management/commands/__init__.py

```diff
@@ -0,0 +1,74 @@
+"""
```

Comment:
@bryanlandia I'm not a fan of actually adding any real code to `__init__.py` modules unless there is some super compelling reason. I'd much rather use a "named" module. I understand in this case any non `__init__.py` files in 'commands' directory implies it is a command, which is a bit of a pain. So I'm not going to be dogmatic about it. I am curious what you think about creating a `management/base.py` module, but won't ask for this for this PR. We can improve this in the future.

Reply:
Interesting.  Yes, I think management/base.py is a good idea.